phpunit.xml.dist

```diff
@@ -7,7 +7,6 @@
          convertErrorsToExceptions="true"
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
-         strict="true"
          bootstrap="vendor/autoload.php">
```

Comment:
should be replaced by `beStrictAboutOutputDuringTests`? everything else is enabled by default I think

Reply:
should probably also be in a separate pr

Reply:
yeah,  seems like enabled by default. guess I reset phpunit to previous state

Reply:
there are a few `beStrict..` options that aren't enabled by default: https://phpunit.readthedocs.io/en/9.5/configuration.html